src/main/java/io/appform/functionmetrics/FunctionTimerAspect.java

```diff
@@ -99,7 +103,10 @@ private MethodData createMethodData(Signature callSignature) {
         final String methodName = Strings.isNullOrEmpty(monitoredFunction.method())
                                   ? callSignature.getName()
                                   : monitoredFunction.method();
-        return new MethodData(className, methodName);
+        final Options options = FunctionMetricsManager.getOptions().orElse(null);
```

Comment:
map to boolean here directly and pass that

Reply:
Done.